### PR TITLE
Contentmanagement api vm broker

### PR DESCRIPTION
**Pull request by akhil-jha**
Capsule installation is failing due to a package.

### CODE REVIEW

**Comment by akhil-jha** [comment]
trigger: test-robottelo
pytest: -k 'test_positive_mirror_on_sync' tests/foreman/api/test_contentmanagement.py

File: tests/foreman/api/test_contentmanagement.py [hunk 1]
```diff
@@ -56,22 +55,22 @@
...
+from robottelo.hosts import Satellite
 from robottelo.utils.issue_handlers import is_open
-from robottelo.vm import VirtualMachine
-from robottelo.vm_capsule import CapsuleVirtualMachine
 
 
-@pytest.fixture(scope='class')
-def capsule_vm():
+@pytest.fixture(scope='function')
+def capsule_vm(capsule_latest):
```
Comment (JacobCallahan) [inline tests/foreman/api/test_contentmanagement.py:63]: This would be great to have under the main broker fixtures!

**Review by JacobCallahan — APPROVED** [review]
Non-blocking suggestion

File: tests/foreman/api/test_contentmanagement.py [hunk 2]
```diff
@@ -949,19 +947,18 @@ def test_positive_mirror_on_sync(self, capsule_vm):
...
-            result = client.run(f'rpm -qa | grep {package_name}')
-
-            assert result.return_code == 0
-            assert package_name in result.stdout[0]
+        rhel7_contenthost.install_katello_ca()
+        rhel7_contenthost.register_contenthost(org.label, activation_key.name)
+        # Install the package
+        package_name = FAKE_1_YUM_REPO_RPMS[2].rstrip('.rpm')
+        result = rhel7_contenthost.run(f'yum install -y {package_name}')
+        assert result.return_code == 0
```
Comment (jameerpathan111) [inline tests/foreman/api/test_contentmanagement.py:955]: I guess it should be like
```suggestion
        assert result.status == 0
```

File: tests/foreman/api/test_contentmanagement.py [hunk 3]
```diff
@@ -949,19 +947,18 @@ def test_positive_mirror_on_sync(self, capsule_vm):
...
+
+        # Ensure package installed
+        result = rhel7_contenthost.run(f'rpm -qa | grep {package_name}')
+
+        assert result.return_code == 0
```
Comment (jameerpathan111) [inline tests/foreman/api/test_contentmanagement.py:960]:
```suggestion
        assert result.status == 0
```

File: tests/foreman/api/test_contentmanagement.py (hunk 1 above)
Comment (akhil-jha) [reply tests/foreman/api/test_contentmanagement.py:63]: got it. Will make a PR for this.

**Review by jameerpathan111 — APPROVED** [review]

**Comment by jameerpathan111** [comment]
@JacobCallahan should we merge this pr or wait for https://github.com/SatelliteQE/robottelo/pull/8660 to be fixed first?

**Review by mshriver — APPROVED** [review]

**Pull request merged**